Dockerfile.ocp

```diff
@@ -4,8 +4,14 @@ COPY . .
 RUN make build
 RUN make tools
 
+COPY ./config/crd/ocp/ocp_kustomization.yaml /go/src/github.com/metal3-io/baremetal-operator/config/crd/kustomization.yaml
```

Comment:
For consistency with STEP 9 I'd keep a RUN cp also here, given that the sources are already available thanks to the previous `COPY . .`. To minimize the layers I'd suggest also to have a single RUN and && the three commands

Reply:
Done.